doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -209,70 +209,99 @@ infrastructure itself from the workloads compute resources.
 <a name="3.2.2"></a>
 ### 3.2.2 Container Networking Services
 
-As all production networking solutions for Kubernetes are based on CNI plugins
-and the implicit requirement in [4.2.2 Virtual Network Interface
-Specifications](../../../ref_model/chapters/chapter04.md#422-virtual-network-interface-specifications)
-that documents the requirement to have the capability to attach several network
-interfaces to the pods, the CNTT architecture must support a CNI metaplugin/CNI
-multiplexer.
+Kubernetes networking is considered an "extension" to the core functionality,
+and is managed through the use of plugins [Network
+Plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/).
+For hardware resources that are needed by Kubernetes applications, [Device
+Plugins](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
```

Comment:
The link is not correct.

Reply:
Thanks - updated in https://github.com/cntt-n/CNTT/pull/2121/commits/d15eeddaa160a783ccf6b5813d409dae7dcef582

Reply:
Is this ok to resolve @CsatariGergely?